cogeo_mosaic/backend/dynamo.py

```diff
@@ -0,0 +1,81 @@
+import functools
+import itertools
+import os
+from typing import Dict, Tuple
+
+import mercantile
+
+from cogeo_mosaic.backend.base import BaseBackend
+
+
+class DynamoDBBackend(BaseBackend):
```

Comment:
Question, is it possible to return the list of `Keys`?

Reply:
That would mean querying the whole table I think? So I would assume no without a performance penalty

Reply:
cool cool

Reply:
Maybe it's possible to do a faster scan (https://stackoverflow.com/a/48734264) to get the key names, but it seems not ideal.

Reply:
So it is true that you don't know if a quadkey exists until you query DynamoDB, but that query is <10ms, so not a big deal

Reply:
that's totally fine 
I was just asking because we use `quakeys` to create geojson somewhere. but again that's optional